hack/ci-e2e-kind-ha.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o nounset
+set -o pipefail
+set -o errexit
+set -x
+
+source $(dirname "${0}")/ci-common.sh
+
+clamp_mss_to_pmtu
+
+# test setup
+make kind-ha-up
+
+# export all container logs and events after test execution
+trap "
+  ( export_logs 'gardener-local-ha';
+    export_events_for_kind 'gardener-local-ha'; export_events_for_shoots )
+  ( make kind-ha-down )
+" EXIT
+
+make KUBECONFIG=$KUBECONFIG gardener-up
```

Comment:
Why are these `KUBECONFIG` vars needed here and below? I don't see this in https://github.com/gardener/gardener/blob/master/hack/ci-e2e-kind.sh, for example?

Reply:
But in the case of make target `ci-e2e-kind-ha-single-zone`, `ci-e2e-kind-ha-multi-zone` initialized as [export KUBECONFIG = $(GARDENER_LOCAL_KUBECONFIG)](https://github.com/gardener/gardener/pull/6719/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R285),  `ci-e2e-kind-ha-single-zone` calls this script [ci-e2e-kind-ha.sh](https://github.com/gardener/gardener/blob/d20707abc044525c42415fdcc034c3d192cf6528/hack/ci-e2e-kind-ha.sh) and internally calls `make gardener-up`

Make target `make gardener-up` which internally refers to `gardener-local` cluster path instead of `gardener-local-ha` path, because  [here](https://github.com/gardener/gardener/blob/master/Makefile#L280) initialized as `export KUBECONFIG = $(GARDENER_LOCAL_KUBECONFIG)`.

So we have to override `KUBECONFIG` var. 
` make KUBECONFIG=$KUBECONFIG gardener-up`